test_ai_disabled.py

```
@@ -0,0 +1,238 @@
+import asyncio
+from unittest.mock import patch, AsyncMock
+import config
+from llm_handler import call_llm_api, call_llm_for_summary, summarize_scraped_content
+from firecrawl_handler import scrape_url_content
+
+def test_config_ai_disabled():
+    """Test that config module's ai_features_enabled behaves correctly."""
+    print("Testing config.ai_features_enabled behavior...")
+    
+    # Test current actual config state
+    print(f"  Current config state:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)